### ISSUE TITLE
[bitnami/external-dns] Two PRs bumped to the same minor version

### ISSUE THREAD

**Issue by mikejoh** [bitnami/charts#15958]
### Name and Version

bitnami/external-dns 6.16.0

### What architecture are you using?

None

### What steps will reproduce the bug?

N/A

### Are you using any custom parameters or values?

N/A

### What is the expected behavior?

N/A

### What do you see instead?

N/A

### Additional information

Not sure if this qualifies as a bug but i noticed the following:

1. On my PR that was merged yesterday i saw that the pipeline that are supposed to publish the chart failed (after merge to `main`), see this comment: https://github.com/bitnami/charts/pull/15804#issuecomment-1494417554
2. Both of these (one of which were mine) PRs bumped to the same minor version of the `external-dns` chart (`6.16.0`): #15804 and #15837. Mine was merged after #15837 which makes this an error on my part, i think so at least, i should've fetched and rebased before that one merged.

The question is, should i create a new PR with a minor bump to `6.17.0` (at the moment)? Any ideas what's going on or is everything working as expected?

**Comment by javsalgar** [comment]
Hi,

Thank you so much for reporting. Could you create a new bump of the patch version so we ensure that all features are released?

**Comment by mikejoh** [comment]
> Hi,
> 
> Thank you so much for reporting. Could you create a new bump of the patch version so we ensure that all features are released?

Sure! Bump PR created, #15968.

**Comment by javsalgar** [comment]
Thank you so much! The team will take a look
